src/Apache.IoTDB/Utils.cs

```diff
@@ -93,5 +94,38 @@ public List<TEndPoint> ParseSeedNodeUrls(List<string> nodeUrls)
             }
             return nodeUrls.Select(ParseTEndPointIpv4AndIpv6Url).ToList();
         }
+
+        public static DateTime ParseIntToDate(int dateInt)
```

Comment:
Can we use `DateTime.TryParseExact` (or `DateOnly.TryParseExact`) to handle integer-to-date conversion?

Reply:
done